Resources/SceneRegistry.gd

```diff
@@ -6,7 +6,8 @@ class_name SceneRegistry extends Node
 # so we're going to minimize how much/often we have to use that until Godot gets proper refactoring :)
 
 const main_scenes = {
-	"StartScreen": "res://Menus/start_screen.gd"
+	"StartScreen": "res://Menus/start_screen.tscn",
```

Comment:
I replaced `.gd` to `.tscn` so I could do the transition from the Credits Screen to the Start screen.